src/app.js

```diff
@@ -61,14 +62,30 @@ if (AppConstants.FXA_ENABLED) {
 }
 
 // disable forced https to allow localhost on Safari
-app.use(
+// Support GA4 per https://developers.google.com/tag-platform/tag-manager/web/csp
+app.use((_req, res, _next) => {
+  res.locals.nonce = crypto.randomBytes(16).toString('hex')
   helmet.contentSecurityPolicy({
     directives: {
-      imgSrc,
-      upgradeInsecureRequests: isDev ? null : []
+      upgradeInsecureRequests: isDev ? null : [],
+      scriptSrc: [
+        "'self'",
+        `'nonce-${res.locals.nonce}'`
+      ],
+      imgSrc: [
+        "'self'",
+        'profile.stage.mozaws.net',
```

Comment:
Oh I see why this was working before - `imgSrc` is created above this and this is handled there, I'll fix this up.